docs/standards/smart-contracts/lsp0-erc725-account.md

```diff
@@ -112,16 +113,21 @@ _Triggers the **[OwnershipTransferred](#ownershiptransferred)** event once the n
 function renounceOwnership() public
 ```
 
-Renounces ownership of the contract in 2 steps. Can only be called by the owner.
+Since renouncing ownership a sensitive operation, it is done as a two step process by calling  `renounceOwnership(..)` twice. First to initiate the process, second as a confirmation.
```

Comment:
```suggestion
Since renouncing ownership is a sensitive operation, it is done as a two-step process by calling  `renounceOwnership(..)` twice. First to initiate the process, second as a confirmation.
```